azerothcore/azerothcore-wotlk
# fix(Core/Spells): Forbid using charge if already charging to someone.

Fixed #6453



## Changes Proposed:
-  Cannot use another charge/intercept if already charging.

**Requires #7152**

## Issues Addressed:

- Closes #6453
- Closes https://github.com/chromiecraft/chromiecraft/issues/917

## Tests Performed:

- No tested

## How to Test the Changes:

1. You need 2 dummies in a straight line Dummy A and Dummy B
2. Set Dummy A as your focus target (target it and type /focus)
3. Target Dummy B, be in Battle Stance and spam this macro, it should be really easy to reproduce the bug


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
